### DISCUSSION TITLE
How to determine the service port 3000?

### DISCUSSION THREAD

**Discussion by irmowan** [ddiu8081/chatgpt-demo#301] [Q&A] [answered]
Hi, I found that the service exposes 3000 port in `Dockerfile`, but I didn't find any code or config that determines this port. Is it astro framework default port? Or How does the framework use this port?

**Comment by ddiu8081** [comment, marked as answer]
Yes, it's Astro's default port. 

Reference: https://docs.astro.build/en/reference/cli-reference/#--port-number
